File: Project_Api/Entities/Benefactor.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Project_Api.Entities
{
    public class Benefactor
    {
        [Key]
        public int Id { get; set; }
        public int RelationshipId { get; set; }
        public int PersonId { get; set; }
        public int SavingsAccountId { get; set; }
        public string Name { get; set; }
        public int Percentage { get; set; }
        public bool Condition { get; set; }
    }
}

File: FrontEnd/src/models/Benefactor.ts
export default class Benefactor{
    id: number | undefined;
    nameDocId: string;
    savingsAccountId: number;
    valueDocIden: string;
    name: string;
    percentage: number;
    user: number | undefined;

    constructor(id: number|undefined,nameDocId: string,savingsAccountId: number,name: string,valueDocIden: string,percentage: number, user: number|undefined){
        this.id = id;
        this.nameDocId = nameDocId;
        this.savingsAccountId = savingsAccountId;
        this.name = name;
        this.valueDocIden = valueDocIden;
        this.percentage = percentage;
        this.user = user;
    }
}